src/test/java/teammates/test/cases/ui/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -35,11 +36,25 @@ public static void classSetup() {
     
     @Test
     public void testAll() throws Exception {
+        testModerationHint();
         testEditResponse();
         testAddResponse();
         testDeleteResponse();
     }
     
+    private void testModerationHint() throws Exception {
+        ______TS("verify moderation hint");
+        
+        submitPage = loginToInstructorEditStudentFeedbackPage("IESFPTCourseinstr", "student1InIESFPTCourse@gmail.tmt",
+                "session1InIESFPTCourse");
+        
+        submitPage.verifyHtmlPart(By.className("navbar"), "/instructorEditStudentFeedbackHintLess.html");
+        submitPage.click(By.id("moderationHintButton"));
```

Comment:
Can I do it in another issue and PR as this issue is related to code quality but has nothing to do with current PR title.